### ISSUE TITLE
We should consider to set a limit rate for accepting new blog posts from our community.

### ISSUE THREAD

**Issue by huan** [wechaty/docusaurus#447]
In the past months, we are great because our community has more and more developers join and they are willing to publish their projects as showcases to the community so that they can be accepted by the Wechaty Developers' Program.

However, this has pros and cons at the same time.

- The good part is that if we have more posts from our developers, then the community will be able to publish more content.
- The bad part is, there will always be true that The-more-the-better is not always true.

So I believe we should set an acceptance rate/number for our blog submissions from the community. For example, we can think about the following three variables for acceptance (per month):

1. The **maximum number** of all the posts that can be accepted from the community
1. The **min/max number in a language** of the posts that can be accepted from the community

For example, we can plan to accept 3-5 posts per month, which should satisfy that no more than 3 in Chinese and no less than 1 in English.

For a summary of the past 30 days (Jul 12 - Aug 12), we have published 21 blog posts, which includes:

- 9 posts from official (SoC, PMC), and
- 12 posts from the community.

To be discussed. @lijiarui

**Comment by lijiarui** [comment]
Good idea! I agree with this, then we can create a better community!

**Comment by huan** [comment]
It's great to know that you agree with me for setting an acceptance rate/number for our blog submissions from the community!

Then I believe we can start from now and let's try to follow the following rules as I mentioned before:

### Community Blog Submission Rules

For each month:

1. **5 (posts)**: The maximum number of all the posts that can be accepted from the community
1. **3 (posts)** The max number of Chinese posts
1. The max number of English posts: no limitation. (as long as the total number is not reached the maximum)

To make it clear what is a community blog post: a community blog submission is a blog post that posted from our non-contributors.

The posts submitted from our contributors, for example, from PMC, SoC, etc, will not fall into the above rules.

### Reviewing & Approvement

We can review and approve the new submissions weekly, for time-saving.

Every week, we pick one best blog posts from the submissions and make it online.

Other submissions will stay in the queue and will wait for the selection in the following week. (In order to improve the probabilities  that it can be accepted, it is better to be improved before next selection).

**Comment by lijiarui** [comment]
Agree, how about we put this here: https://github.com/wechaty/wechaty.js.org

Then anyone who pr can see this.

**Comment by huan** [comment]
Of course. Please feel free to add them, and let's continue thinking and trying to improve them in the following days.

For example, I think there might need another rule to make sure we will not be flooding by the Pull Requests:

4. When the community blog submission queue length is > 9, we will only keep the TOP9 PRs sorted by the quality. The blog submission that not in the TOP9  will be closed so that we can make sure our PR list is not too long. The PR can be recreated if the submitter want, however, it's quality should be improved compared to the last PR, or it will most likely not be able to be merged.
